lib/client.js

```diff
@@ -45,6 +45,7 @@ extend(Raven.prototype, {
     this.name = options.name || process.env.SENTRY_NAME || require('os').hostname();
     this.root = options.root || process.cwd();
     this.transport = options.transport || transports[this.dsn.protocol];
+    this.sendTimeout = options.sendTimeout || 1;
```

Comment:
would we ever be more defensive about types here? I could imagine something that checks for a number less than 1000 but we don't do that anywhere else in options